src/components/TodoFilter.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import classnames from 'classnames';
+import { TodoStatus } from '../types/TodoStatus';
+import { SearchLink } from './SearchLink';
+
+interface Props {
+  status: string
+  itemsLeft: number
+  itemsCompleted: number
+  clearCompletedTodos: () => void
+}
+
+export const TodoFilter: React.FC<Props> = ({
+  status,
+  itemsLeft,
+  itemsCompleted,
+  clearCompletedTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {status === TodoStatus.Completed
+          ? (`${itemsCompleted} items completed`)
+          : (`${itemsLeft} items left`)}
+      </span>
+
+      <nav className="filter">
+        <SearchLink
+          params={{ status: null }}
+          className={
+            classnames(
+              'filter__link',
+              { selected: !status },
+            )
+          }
+        >
+          All
+        </SearchLink>
+
+        <SearchLink
+          params={{ status: 'active' }}
+          className={
+            classnames(
+              'filter__link',
+              { selected: status === TodoStatus.Active },
+            )
+          }
+        >
+          Active
+        </SearchLink>
+
+        <SearchLink
+          params={{ status: 'completed' }}
```

Comment:
Replace the hardcoded string 'completed' with the constant `TodoStatus.Completed` for consistency.